modules/@apostrophecms/doc-type/index.js

```diff
@@ -328,6 +328,15 @@ module.exports = {
           }
         }
       },
+      afterDelete: {
+        async deleteRelatedReverseId(req, doc) {
+          // When deleting a draft document, we remove related reverse IDs
+          // of documents having a relation to the deleted one
+          if (doc.aposMode === 'draft') {
```

Comment:
Hmm. What about localized: false types?

Reply:
Still a draft PR ahah but `localized: false` documents are never deleted, they can only be archived.

Reply:
True right now via the UI, but the delete backend API exists and someone may invoke it.

Reply:
Right I added support for doc not having aposMode (aka unlocalized docs)